app/survey_config/business_config.py

```diff
@@ -18,7 +21,7 @@ class BusinessSurveyConfig(
     footer_links: Iterable[MutableMapping] = field(
         default_factory=lambda: [
             Link(lazy_gettext("What we do"), "#").__dict__,
-            Link(lazy_gettext("Contact us"), "#").__dict__,
+            Link(lazy_gettext("Contact us"), SurveyConfig.contact_us_url).__dict__,
```

Comment:
Needs to the property from `self`, for which you need to make use of `__post_init__`. The same applies to the footer legal links.